src/Sulu/Bundle/AdminBundle/Resources/js/components/DatePicker/DatePicker.js

```diff
@@ -0,0 +1,71 @@
+// @flow
+import React from 'react';
+import ReactDatetime from 'react-datetime';
+import 'react-datetime/css/react-datetime.css';
+import type Moment from 'moment';
+import moment from 'moment';
+import Input from '../Input';
+import './datePicker.scss';
+
+type Props = {
+    value?: string | Moment,
+    placeholder?: string,
+    onChange: (value: string) => void,
+    options: Object,
+};
+
+export default class DatePicker extends React.PureComponent<Props> {
+    static defaultProps = {
+        options: {},
+    };
+
+    handleChange = (date: string | Moment) => {
+        this.props.onChange(date);
+    };
+
+    renderInput = (props: Object, openCalendar: () => void) => {
+        return (
+            <Input {...props} icon="calendar" onChange={this.handleChange} onIconClick={openCalendar} />
+        );
+    };
+
+    render() {
+        const {placeholder} = this.props;
+        let {value} = this.props;
+        let {options} = this.props;
+        let inputProps = {};
+
+        if (options) {
+            if (!options.hasOwnProperty('closeOnSelect')) {
+                options.closeOnSelect = true;
+            }
+
+            if (!options.hasOwnProperty('timeFormat')) {
+                options.timeFormat = false;
+            }
+
+            if (!options.hasOwnProperty('dateFormat')) {
+                options.dateFormat = moment.localeData().longDateFormat('L');
+            }
+        }
+
+        if (placeholder) {
+            inputProps.placeholder = placeholder;
+        }
+
+        if (!inputProps.placeholder) {
+            inputProps.placeholder = options.dateFormat +
+                (options.timeFormat ? ' ' + moment.localeData().longDateFormat('LT') : '');
+        }
```

Comment:
The above two ifs can also be simplified. Either this way:

```javascript
inputProps.placeholder = placeholder;

if (!inputProps.placeholder) {
    // ...
}
```
Or my preferred solution, because it is even shorter:

```javascript
inputProps.placeholder = placeholder
    ? placeholder
    : options.dateFormat + /* ... */;
```

Another option would be to use the `||` operator:

```javascript
inputProps.placeholder = placeholder || options.dateFormat + /* ... */;
```